src/MockBot.Api/Interfaces/IChatService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using MockBot.Api.Models;
+
+namespace MockBot.Api.Interfaces
+{
+    public interface IChatService
+    {
+        public Chat CreateChat();
+
+        public IEnumerable<Chat> FindAll();
```

Comment:
super minor and feel free to ignore. I'm not sure that `FindAll` is the right name for this as it indicates that you are able to search but this actually just returns all `Chat` objects. For me `GetAll` would make more sense.

Reply:
This is a Java thing, find can return no result, get should always return something. I've changed this maybe twice now, based on who's looking. If there's a specific style guide, I'd like to follow that.

Reply:
OK, in that case, go with what you think feels right and we can address this later as we define conventions for the project. I'll resolve the issue